ui/page/livestreamOverlay/view.jsx

```diff
@@ -0,0 +1,301 @@
+// @flow
+import * as React from 'react';
+import Card from 'component/common/card';
+import Page from 'component/page';
+import ChannelSelector from 'component/channelSelector';
+import Spinner from 'component/spinner';
+import SettingsRow from 'component/settingsRow';
+import { FormField } from 'component/common/form';
+
+const DEBOUNCE_REFRESH_MS = 1000;
+
+// ****************************************************************************
+// ****************************************************************************
+
+type Props = {
+  activeChannelClaim: ChannelClaim,
+  settingsByChannelId: { [string]: PerChannelSettings },
+  fetchCreatorSettings: (channelId: string) => void,
+};
+
+export default function SettingsOverlayPage(props: Props) {
+  const { activeChannelClaim, settingsByChannelId, fetchCreatorSettings } = props;
+
+  const [lastUpdated] = React.useState(1);
+
+  // **************************************************************************
+  // **************************************************************************
+
+  // Re-sync list on first idle time; mainly to correct any invalid settings.
+  React.useEffect(() => {
+    if (lastUpdated && activeChannelClaim) {
+      const timer = setTimeout(() => {
+        fetchCreatorSettings(activeChannelClaim.claim_id);
+      }, DEBOUNCE_REFRESH_MS);
+      return () => clearTimeout(timer);
+    }
+  }, [lastUpdated, activeChannelClaim, fetchCreatorSettings]);
+
+  // **************************************************************************
+  // **************************************************************************
+
+  const isBusy =
+    !activeChannelClaim || !settingsByChannelId || settingsByChannelId[activeChannelClaim.claim_id] === undefined;
+  const isDisabled =
+    activeChannelClaim && settingsByChannelId && settingsByChannelId[activeChannelClaim.claim_id] === null;
+
+  // **************************************************************************
+  // **************************************************************************
+  return (
+    <Page
+      noFooter
+      backout={{ title: __('Overlay settings and instructions'), backLabel: __('Back') }}
+      className="card-stack"
+    >
+      <div className="card-stack">
+        <ChannelSelector hideAnon />
+
+        {isBusy && (
+          <div className="main--empty">
+            <Spinner />
+          </div>
+        )}
+
+        {isDisabled && (
+          <Card
+            title={__('Settings unavailable for this channel')}
+            subtitle={__("This channel isn't staking enough LBRY Credits to enable Overlay settings and instructions.")}
+          />
+        )}
```

Comment:
This part doesn't make sense ... `isDisabled` is true when we have not fetches the per-user settings, no it has nothing to do with staking.  Not sure what the intention is here ... either check the stake level, or remove the subtitle.

Reply:
This is no longer there.